docs/CompatibleSoftware.rst

```diff
@@ -66,58 +66,121 @@ OCIO is natively integrated in
 Full support is provide for both image import/export, as well as image display.
 
 
+Katana
```

Comment:
Can we reorder the projects alphabetically so they are easier to locate if someone is looking for one in particular? I would sort them by the project name; not the company name (i.e. "Autodesk Maya" is treated as "M", not "A").